src/main/scala/cromwell/Main.scala

```diff
@@ -74,11 +74,9 @@ object Main extends App {
 
     import PromiseActor.EnhancedActorRef
 
-    val promise = runner.askNoTimeout(RunWorkflow)
-    waitAndExit(promise, CromwellSystem)
+    waitAndExit(runner.askNoTimeout(RunWorkflow), CromwellSystem)
```

Comment:
ToL: Is there a difference here, or just a preference to have more functions on one line?